### ISSUE TITLE
Learning rate degeneracy

### ISSUE THREAD

**Issue by mmaupas** [sachaguer/jaxili#42]
There is a degeneracy in the definition of the learning rate for `NLE` and `NPE`.

`NDE.train( learning_rate=1e-6, ... )` sets the learning rate to `1e-6`.

But
`optimizer_hparams = { 'lr': 1e-6, ... }`
`NDE.train( ... )`
keeps the learning rate at the default value `5e-4`.

**Comment by sachaguer** [comment]
I checked the code. I have to fix the documentation because  `NDE.train( ... )` does not take `optimizer_hparams` as argument.
I will add the argument that can be given to the function `train` and their default value. This include `optimizer_haparams` that are read in the kwargs.

**Comment by sachaguer** [comment]
Added a warning if the user if using `optimizer_hparams` in the `kwargs`.
